### PR TITLE
Feature/xivy 5006 msgraph

### PR DESCRIPTION
**Pull request by ivy-rew**

### CODE REVIEW

File: market/connector/microsoft365/demo/msGraphDemo/pom.xml [hunk 1]
```diff
@@ -38,7 +38,9 @@
           <showErrors>true</showErrors>
           <reportsDirectory>${java.io.tmpdir}</reportsDirectory>
           <properties>
-            <openapi.url>https://raw.githubusercontent.com/microsoftgraph/microsoft-graph-openapi/master/v1.0.json</openapi.url>
+            <openapi.url><![CDATA[
+              https://graphexplorerapi.azurewebsites.net/openapi?tags=me.user,me.calendar,users.calendar&openApiVersion=2&graphVersion=v1.0&format=json
```
Comment (ivy-lli) [inline market/connector/microsoft365/demo/msGraphDemo/pom.xml:42]: Why not use openApiVersion=3?

File: market/connector/microsoft365/demo/msGraphDemoTest/src/com/microsoft/graph/json/calendarView.json [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+                },
+                {
+                    "type": "required",
+                    "status": {
+                        "response": "accepted",
+                        "time": "0001-01-01T00:00:00Z"
+                    },
+                    "emailAddress": {
+                        "name": "Lukas Lieb",
+                        "address": "lukas.lieb@mailinator.com"
```
Comment (ivy-lli) [inline market/connector/microsoft365/demo/msGraphDemoTest/src/com/microsoft/graph/json/calendarView.json:81]: lukas.lieb@mailinator.com didn't know about this email address 🤣

File: market/connector/microsoft365/meta.json [hunk 3]
```diff
@@ -21,7 +21,7 @@
 		{
 			"id": "rest-client",
 			"data": {
-				"openApiUrl": "https://raw.githubusercontent.com/microsoftgraph/microsoft-graph-openapi/master/v1.0.json",
+				"openApiUrl": "https://graphexplorerapi.azurewebsites.net/openapi?tags=me.calendar,me.user,users.calendar&openApiVersion=2&graphVersion=v1.0&format=json",
```
Comment (ivy-lli) [inline market/connector/microsoft365/meta.json:24]: openapi version 3?

**Review by ivy-lli — APPROVED** [review]
Cool stuff. Is there a reason why you use openapi in version 2 and not 3?

File: market/connector/microsoft365/demo/msGraphDemo/pom.xml (hunk 1 above)
Comment (ivy-rew) [reply market/connector/microsoft365/demo/msGraphDemo/pom.xml:42]: with openAPI2 we have support for 'inherited' types and swagger-codegen can reflect that perfectly to Java. 

E.g. the api describes an outlookt event with hierarchy 
graphEntity -> outlookObject -> event

but if we use openapi3 ... all the fields/attributes from parent objects (graphEntity, outlookObject) are not reflected unto the domain model. 

I don't know whether this is an issue how microsoft slices the services, or whether oas2 has better java support. But for now it seems to be the better choice to use swagger2 specs.

**Pull request merged**